src/app/App.js

```diff
@@ -0,0 +1,43 @@
+import {useState} from 'react';
+import './App.css';
+import Counter from '../shared/ui/Counter/Counter';
+import SearchForm from '../shared/ui/SearchForm/SearchForm';
+import SelectTabs from '../shared/ui/SelectTabs/SelectTabs';
+
+
+const genres = [
```

Comment:
mocks should not be a part of ui dumb components, let's put these `genres` array  near `SelectTabs.js` for now: 

<img width="182" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/b33b6db7-e57d-4999-ae87-900b1bfa4491">

we will delete this file when API requests are implemented

Reply:
done